instrument/devices/damm.py

```diff
@@ -0,0 +1,43 @@
+"""
+Damm in station 8-ID-D
+"""
+
+__all__ = """
+    damm
+""".split()
+
+import logging
+
+logger = logging.getLogger(__name__)
+logger.info(__file__)
+
+from ophyd import Component as Cpt
+from ophyd import Device
+from ophyd import FormattedComponent as FCpt
+from ophyd import EpicsMotor
+
+
+class Slit2(Device):
+    def __init__(
+        self,
+        prefix: str,
+        h_motor: str,
+        v_motor: str,
+        *args,
+        **kwargs,
+    ):
+        # Determine the prefix for the motors
+        pieces = prefix.strip(":").split(":")
+        self.motor_prefix = ":".join(pieces[:-1])
+
+        self._h_motor = h_motor
```

Comment:
Why name these h & v when these are pitch and yaw motor names?

Reply:
because that is as it was written on the epics screen used by 8-ID

Reply:
Ooph!